independent-publisher-connectors/PowerBI/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1062 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Default title",

```

Comment:
@paulculmsee Can you please provide valid title for the connector ?

Documentation - [Title of connector](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#title)